wordpress-mobile/WordPress-iOS
# UL&S: SiteAddressViewController scrolling bug

Fixes # n/a
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/319

This PR is a quick way to see and try out the scroll bug. None of the things I tried fixed the issue. 

1. Removing `NUXViewControllerBase` [tap gesture](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/blob/develop/WordPressAuthenticator/NUX/NUXViewControllerBase.swift#L122-L128) that dismisses the keyboard
2. Using `NUXViewControllerBase` tap gesture and setting `tgr.cancelsTouchesInView = false`

Then I thought maybe the top autolayout constraint set to a safe area was causing the problem. That didn't fix the issue either. I thought it might have something to do with the large titles, but my searches aren't returning any results related to my problem.

### To test:
1. Set your simulator to a small screen, like iPhone SE 2nd gen.
2. Check out this branch, `rake dependencies`, build and run.
3. Try to make the table scroll after an error message appears.

| Before | After |
| --- | --- |
| ![siteaddressvc-scroll-bug-original](https://user-images.githubusercontent.com/1062444/86956294-79828d00-c11e-11ea-9d27-fcae50fc8f8a.gif) | ![siteaddressvc-scroll-bug](https://user-images.githubusercontent.com/1062444/86956988-a6836f80-c11f-11ea-80de-ca18829ddfe1.gif) |

PR submission checklist:
